classes/submission/SupplementaryFile.inc.php

```diff
@@ -0,0 +1,244 @@
+<?php
+
+/**
+ * @file classes/submission/SupplementaryFile.inc.php
+ *
+ * Copyright (c) 2014-2015 Simon Fraser University Library
+ * Copyright (c) 2003-2015 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class SupplementaryFile
+ * @ingroup submission
+ * @see SubmissionFileDAO
+ *
+ * @brief Supplementary file class.
+ */
+
+import('lib.pkp.classes.submission.SubmissionFile');
+
+class SupplementaryFile extends SubmissionFile {
+	/** @var array image file information */
+	var $_imageInfo;
+
+	/**
+	 * Constructor
+	 */
+	function SupplementaryFile() {
+		parent::SubmissionFile();
+	}
+
+
+	//
+	// Getters and Setters
+	//
+	/**
+	 * Get "localized" creator (if applicable).
+	 * @param $preferredLocale string
+	 * @return string
+	 */
+	function getLocalizedCreator($preferredLocale = null) {
+		return $this->getLocalizedData('creator', $preferredLocale);
+	}
+
+	/**
+	 * Get creator.
+	 * @param $locale
+	 * @return string
+	 */
+	function getCreator($locale) {
+		return $this->getData('creator', $locale);
+	}
+
+	/**
+	 * Set creator.
+	 * @param $creator string
+	 * @param $locale
+	 */
+	function setCreator($creator, $locale) {
+		return $this->setData('creator', $creator, $locale);
+	}
+
+	/**
+	 * Get localized subject
+	 * @return string
+	 */
+	function getLocalizedSubject() {
+		return $this->getLocalizedData('subject');
+	}
+
+	/**
+	 * Get subject.
+	 * @param $locale string
+	 * @return string
+	 */
+	function getSubject($locale) {
+		return $this->getData('subject', $locale);
+	}
+
+	/**
+	 * Set subject.
+	 * @param $subject string
+	 * @param $locale string
+	 */
+	function setSubject($subject, $locale) {
+		return $this->setData('subject', $subject, $locale);
+	}
+
+	/**
+	 * Get localized description
+	 * @return string
+	 */
+	function getLocalizedDescription() {
+		return $this->getLocalizedData('description');
+	}
+
+	/**
+	 * Get file description.
+	 * @param $locale string
+	 * @return string
+	 */
+	function getDescription($locale) {
+		return $this->getData('description', $locale);
+	}
+
+	/**
+	 * Set file description.
+	 * @param $description string
+	 * @param $locale string
+	 */
```

Comment:
I think you used the get* method as a template... So you have to remove the return from all set* methods, I think.